src/Destructurama.FSharp/Destructurama.FSharp.fsproj

```diff
@@ -29,10 +29,10 @@
     </Description>
     <Copyright>Copyright 2019 Destructurama Contributors, Serilog Contributors</Copyright>
     <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
-    <PackageIconUrl>http://destructurama.github.io/pages/images/destructurama.png</PackageIconUrl>
+    <iconUrl>http://destructurama.github.io/pages/images/destructurama.png</iconUrl>
```

Comment:
Don't think this is the right fix here - the replacement for `PackageIconUrl` is `PackageIcon` - HTH

Reply:
You're right, proper fix here would be to include the icon in the project with `<None Include="destructurama.png" PackagePath="destructurama.png" />` and then set `<PackageIcon>destructurama.png</PackageIcon>`

Reply:
Sorry, that was my mistake... For what it's worth, I agree with your proposal.